custom_components/open_epaper_link/imagegen.py

```diff
@@ -60,6 +60,9 @@ def getIndexColor(color):
     else:
         return white
 
+def should_show_element(element):
+    return element['visible'] if 'visible' in element else True
```

Comment:
Is there a reason to not use the more pythonic `return element.get('visible', True)`?

Reply:
I'm really just haven't used python much. But the most of the code in this file doesn't use .get() either so I was trying to be consistant. For example:
```python
            if not "color" in element:
                color = "black"
            else: 
                color = element['color']
```